src/components/stable/gux-modal/gux-modal.tsx

```
@@ -146,6 +149,22 @@ export class GuxModal {
     const dismissEvent = this.guxdismiss.emit();
     if (!dismissEvent.defaultPrevented) {
       this.root.remove();
+      this.triggerElement?.focus();
```

Should we provide a helper method of devs that will be preventing the default and hiding the modal?
```
  @Method()
  async guxFocusTriggerElement(): Promise<void> {
    this.triggerElement?.focus();
  }
```
Would also have to be shown in the examples.

---

I will add this

---

This might be a bit more complicated than I thought. I will discuss it with you today.